.github/workflows/deploy.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy SNAPSHOT
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "The branch to use to deploy a SNAPSHOT from."
+        required: true
+        default: "main"
+jobs:
+  deploy:
+    name: Deploy SNAPSHOT to Sonatype
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        ref: ${{ github.event.inputs.branch }}
+    - name: Set up JDK
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'adopt'
+        java-version: '8'
+        # this creates a settings.xml with the following server
+        settings-path: ${{ github.workspace }}
+        server-id: ossrh # Value of the distributionManagement/repository/id field of the pom.xml
```

Comment:
Some comments start with upper case and others with lower case

Reply:
OMG